etc/api.conf

```diff
@@ -6,15 +6,19 @@
 #   disabletest <testname>
 #     Where:
 #       <testname> is updated, gc, geo, whitelist, or check
+#   updated-slowrepos <fully.qualified.repository.name>
```

Comment:
I'm sorry I wasn't clear enough.  I meant to only change the internal variable name to be plural.  The user-facing keyword should still be singular `updated-slowrepo` because it only adds one repository.